src/Application/Regulation/Query/GetRegulationsQueryHandler.php

```diff
@@ -29,12 +31,12 @@ public function __invoke(GetRegulationsQuery $query): Pagination
         foreach ($rows['items'] as $row) {
             $locationView = null;
 
-            if ($row['location']) {
-                [$roadName, $cityLabel, $cityCode] = explode('#', $row['location']);
-                $locationView = new LocationView($cityCode, $cityLabel, $roadName);
-            } elseif ($row['departmentalRoad']) {
-                [$roadNumber, $administrator] = explode('#', $row['departmentalRoad']);
-                $locationView = new LocationView(null, null, null, $roadNumber, $administrator);
+            if ($row['namedStreet']) {
+                [$roadName, $cityLabel, $cityCode] = explode('#', $row['namedStreet']);
+                $locationView = new NamedStreetView(RoadTypeEnum::LANE->value, $cityCode, $cityLabel, $roadName);
```

Comment:
cf mon commentaire sur NamedStreetView

```suggestion
                $locationView = new NamedStreetView($cityCode, $cityLabel, $roadName);
```